api/src/test/java/org/openmrs/validator/VisitValidatorTest.java

```diff
@@ -448,4 +448,52 @@ public void validate_shouldFailValidationIfFieldLengthsAreNotCorrect() {
 		new VisitValidator().validate(visit, errors);
 		assertEquals(true, errors.hasFieldErrors("voidReason"));
 	}
+	
+	/**
+	 * @see VisitValidator#validate(Object,Errors)
+	 */
+	@Test
+	public void validate_shouldFailValidationIfVisitDateBeforeBirthDate() {
+		VisitService vs = Context.getVisitService();
+		Visit visit = new Visit();
+		
+		visit.setVisitType(vs.getVisitType(1));
+		visit.setPatient(Context.getPatientService().getPatient(2));
+		Calendar c = Calendar.getInstance();
+		c.set(1974, 4, 8);
+		visit.setStartDatetime(c.getTime());
+		
+		Errors errors = new BindException(visit, "visit");
+		new VisitValidator().validate(visit, errors);
+		assertTrue(errors.hasFieldErrors("startDatetime"));
+	}
+	
+	/**
+	 * @see VisitValidator#validate(Object,Errors)
+	 */
```

Comment:
remove these javadocs as they do not provide value. we should not add them anymore. you can easily navigate or see javadocs of the method under test in the test itself